client/app/components/composites/SearchBar/SearchBar.css

```diff
@@ -390,4 +322,28 @@
   @media (--large-viewport) {
     display: none;
   }
+
+}
+
+/* CSS triangle pointing to the search toggle icon */
+.mobileToggleArrow {
+  display: none;
+  position: absolute;
+  bottom: -1px;
+  right: calc(var(--SearchBar_mobileHeight) / 2 - var(--SearchBar_mobileTriangleSize));
+  border: var(--SearchBar_mobileTriangleSize) solid transparent;
+
+  /* We must adjust the brightness here to match either the input
+     background or the submit button background when the toggle
+     position changes in the breakpoint. This cannot be done in JS by
+     switching the color since the breakpoints are unknown there. */
+  filter: brightness(80%);
```

Comment:
One possible solution would be to create two separate arrows that have bg color assigned in JS, and the CSS would just swap between those in the breakpoint.